wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -1386,7 +1382,9 @@ public final class MLSService: MLSServiceInterface {
                         logger.error("failed to commit pending proposals: \(String(describing: error))")
                     }
                 }
+
             }
+            await taskGroup.waitForAll()
```

Comment:
Oh I thought a `TaskGroup` was always waiting for all tasks to complete before returning.

"A group waits for all of its child tasks to complete or be canceled before it returns. After this function returns, the task group is always empty."

https://developer.apple.com/documentation/swift/withtaskgroup(of:returning:body:)

Reply:
it doesn't wait unless you invoke `waitForAll()`?

Reply:
discussed with @markusfassbender and updated https://github.com/wireapp/wire-ios/pull/1153/commits/40b46ed2b876d8adcba28f8d9a5ca591d6a5d88e